src/esp32/analog.rs

```diff
@@ -1,8 +1,8 @@
 #![allow(dead_code)]
 use crate::common::analog::AnalogReader;
+use crate::esp_idf_svc::hal::adc::{AdcChannelDriver, AdcDriver};
```

Comment:
Within the micro-rdk proper, we obtain `esp-idf-svc` and its re-exported `sys` and `hal` crates via `crate::`.